cdap-e2e-tests/src/e2e-test/java/io/cdap/cdap/stepsdesign/TestSetupHooks.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.stepsdesign;
+
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.StorageException;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.e2e.utils.StorageClient;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.UUID;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import stepsdesign.BeforeActions;
+
+/**
+ * Setup gcs for Hub tests.
+ */
+public class TestSetupHooks {
+  public static String gcsSourceBucketName = StringUtils.EMPTY;
+  public static String gcsTargetBucketName = StringUtils.EMPTY;
+
+  @Before(order = 1, value = "@GCS_SOURCE_TEST")
+  public static void createBucketWithCSVFile() throws IOException, URISyntaxException {
+    gcsSourceBucketName = createGCSBucketWithFile(PluginPropertyUtils.pluginProp("testFile"));
+    PluginPropertyUtils.addPluginProp("gcsSourceBucket", "gs://" + gcsSourceBucketName + "/"  +
+            PluginPropertyUtils.pluginProp("testFile"));
+    BeforeActions.scenario.write("GCS source bucket1 name - " + gcsSourceBucketName);
+  }
+  private static String createGCSBucketWithFile(String filePath)
+      throws IOException, URISyntaxException {
+    String bucketName = StorageClient.createBucket("e2e-test-" + UUID.randomUUID()).getName();
```

Comment:
As we are not going to use it as source kindly remove it.

Reply:
removed.